packages/integrations/netlify/src/index.ts

```diff
@@ -246,6 +275,18 @@ export default function netlifyIntegration(
 		}
 	}
 
+	async function getFilesByGlob(
+		include: Array<string> = [],
+		exclude: Array<string> = [],
+	): Promise<Array<URL>> {
+		const files = await glob(include, {
+			cwd: fileURLToPath(rootDir),
+			absolute: true,
+			ignore: exclude,
+		});
+		return files.map((file) => new URL(pathToFileURL(file)));
```

Comment:
```suggestion
		return files.map((file) => pathToFileURL(file));
```
`pathToFileURL` already return a URL object